### ISSUE TITLE
Convert to Tensorrt

### ISSUE THREAD

**Issue by congthuyen94** [mk-minchul/AdaFace#31]
Hi, thank you for your excellent work!
I want to deploy this model to Tensorrt. Could you give me some guidelines

**Comment by TorRient** [comment]
Fisrt step: remove norm output at forward function in net.py before convert
Second step: convert onnx -> tensorRT
`import numpy as np
import onnx
import torch


def convert_onnx(net, path_module, output, opset=11, simplify=False):
    assert isinstance(net, torch.nn.Module)
    img = np.random.randint(0, 255, size=(112, 112, 3), dtype=np.int32)
    img = img.astype(np.float)
    img = (img / 255. - 0.5) / 0.5  # torch style norm
    img = img.transpose((2, 0, 1))
    img = torch.from_numpy(img).unsqueeze(0).float()

    statedict = torch.load(path_module)['state_dict']
    # net.load_state_dict(weight, strict=True)
    model_statedict = {key[6:]:val for key, val in statedict.items() if key.startswith('model.')}
    net.load_state_dict(model_statedict)
    net.eval()
    torch.onnx.export(net, img, output, input_names=["input"], output_names=["output"], keep_initializers_as_inputs=False, verbose=False, opset_version=opset)
    model = onnx.load(output)
    graph = model.graph
    graph.input[0].type.tensor_type.shape.dim[0].dim_param = '?'
    if simplify:
        from onnxsim import simplify
        model, check = simplify(model)
        assert check, "Simplified ONNX model could not be validated"
    onnx.save(model, output)

    
if __name__ == '__main__':
    import os
    import argparse
    from model_irse_adaface import build_model

    parser = argparse.ArgumentParser(description='ArcFace PyTorch to onnx')
    parser.add_argument('input', type=str, help='input backbone.pth file or path')
    parser.add_argument('--output', type=str, default=None, help='output onnx path')
    parser.add_argument('--network', type=str, default=None, help='backbone network')
    parser.add_argument('--simplify', type=bool, default=False, help='onnx simplify')
    args = parser.parse_args()
    input_file = args.input
    if os.path.isdir(input_file):
        input_file = os.path.join(input_file, "model_ir50_adaface_web600k.pt")
    assert os.path.exists(input_file)
    # model_name = os.path.basename(os.path.dirname(input_file)).lower()
    # params = model_name.split("_")
    # if len(params) >= 3 and params[1] in ('arcface', 'cosface'):
    #     if args.network is None:
    #         args.network = params[2]
    assert args.network is not None
    print(args)
    backbone_onnx = build_model(args.network)
    convert_onnx(backbone_onnx, input_file, args.output, simplify=args.simplify)`
refer: https://github.com/deepinsight/insightface/blob/master/recognition/arcface_torch/torch2onnx.py
